ignite/cmd/plugin.go

```diff
@@ -542,30 +543,26 @@ func NewPluginDescribe() *cobra.Command {
 	}
 }
 
-func printPlugins(session *cliui.Session) error {
+func printPlugins(session *cliui.Session) {
 	var entries [][]string
-	for _, p := range plugins {
-		var status string
+	buildStatus := func(p *plugin.Plugin) string {
 		if p.Error != nil {
-			status = fmt.Sprintf("❌ Error: %v", p.Error)
-		} else {
-			manifest, err := p.Interface.Manifest()
-			if err != nil {
-				return fmt.Errorf("error while loading plugin manifest: %w", err)
-			}
-
-			var (
-				hookCount = len(manifest.Hooks)
-				cmdCount  = len(manifest.Commands)
-			)
-			status = fmt.Sprintf("✅ Loaded 🪝 %d 💻 %d", hookCount, cmdCount)
+			return fmt.Sprintf("%s Error: %v", icons.NotOK, p.Error)
 		}
-		entries = append(entries, []string{p.Path, status})
+		manifest, err := p.Interface.Manifest()
+		if err != nil {
+			return fmt.Sprintf("%s Error: Manifest() returned %v", icons.NotOK, err)
+		}
+		var (
+			hookCount = len(manifest.Hooks)
+			cmdCount  = len(manifest.Commands)
+		)
+		return fmt.Sprintf("%s Loaded 🪝%d 💻%d", icons.OK, hookCount, cmdCount)
+	}
+	for _, p := range plugins {
+		entries = append(entries, []string{p.Path, buildStatus(p)})
 	}
-
 	session.PrintTable([]string{"Path", "Status"}, entries...)
```

Comment:
maybe should we handle this error?

Reply:
you're right, fixed here [4c58511](https://github.com/ignite/cli/pull/3240/commits/4c585114bf4d413d04d058ae03b6150a37bfe4b7).